docs/source/delta-drop-feature.md

```diff
@@ -27,6 +27,7 @@ You can drop the following Delta table features:
 - `deletionVectors`. See [_](delta-deletion-vectors.md).
 - `typeWidening-preview`. See [_](delta-type-widening.md). Type widening is available in preview in <Delta> 3.2.0 and above.
 - `v2Checkpoint`. See [V2 Checkpoint Spec](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#v2-spec). Drop support for V2 Checkpoints is available in <Delta> 3.1.0 and above.
+- `coordinatedCommits-preview`. See [_](delta-coordinated-commit.md). Coordinated Commits is available in preview in <Delta> 4.0.
```

Comment:
The file name is "delta-coordinated-commits.md" and not "delta-coordinated-commit.md" ?